auditlog/management/commands/auditlogmigratejson.py

```diff
@@ -72,17 +88,25 @@ def _apply_django_migration(_logs) -> int:
             import json
 
             updated = []
+            errors = []
             for log in _logs:
                 try:
                     log.changes = json.loads(log.changes_text)
                 except ValueError:
-                    self.stderr.write(
-                        f"ValueError was raised while migrating the log with id {log.id}."
-                    )
+                    errors.append(log.id)
                 else:
                     updated.append(log)
 
             LogEntry.objects.bulk_update(updated, fields=["changes"])
+            if errors:
+                self.stderr.write(
+                    self.style.ERROR(
+                        f"ValueError was raised while converting the the logs with these ids into json."
```

Comment:
Typo: `the the logs`

Reply:
fixed

Reply:
@aleh-rymasheuski sorry I forgot to say thank you. I've been under a lot of stress lately.